src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -138,6 +138,8 @@
     private static final Allele FAKE_REF_ALLELE = Allele.create("N", true); // used in isActive function to call into UG Engine. Should never appear anywhere in a VCF file
     private static final Allele FAKE_ALT_ALLELE = Allele.create("<FAKE_ALT>", false); // used in isActive function to call into UG Engine. Should never appear anywhere in a VCF file
 
+    public static final int MINIMUM_READ_LENGTH = 10;
```

Comment:
This variable name needs to be more explicit, since there are a lot of points in the code where one might want to enforce a minimum read length.  In `Mutect2Engine` it's `MINIMUM_READ_LENGTH_AFTER_TRIMMING`, which is good.  But, instead of duplicating the constant, you should put a single constant that both `Mutect2` and `HaplotypeCaller` can use.  `AssemblyBasedCallerUtils` would be a reasonable place to put this constant.

Reply:
Wait. . . I'm confused, isn't this the code change for issue #5079?  (The comment still applies, though).

Reply:
Yes, it is, sorry. It was part of the testing that I did for #5079 that I hadn't removed.

Reply:
👍 Changed it